docs/current/sailbot_workspace/setup.md

```diff
@@ -171,31 +198,34 @@ git clone https://github.com/UBCSailbot/sailbot_workspace.git
 ## 5. Open Sailbot Workspace in a Dev Container
 
 1. Ensure that Docker is running
-2. Run the `Dev Containers: Reopen in Container` command in the
-   [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
+2. Run the `Dev Containers: Reopen in Container` VS Code command
 
-## 6. Open the VS Code workspace file
+## 6. Open the workspace file
 
 1. Open the file `.devcontainer/config/sailbot_workspace.code-workspace` in VS Code
 2. Click `Open Workspace`
 
-## 7. Run the `setup` VS Code task
+## 7. Run the `setup` task
 
 !!! info ""
 
     Moved from Run to Setup page in [:octicons-tag-24: v1.1.0](https://github.com/UBCSailbot/sailbot_workspace/releases/tag/v1.1.0){target=_blank}
 
-The `setup` task clones the repositories defined in [`src/new_project.repos`](https://github.com/UBCSailbot/sailbot_workspace/blob/main/src/new_project.repos){target=_blank}
-and updates dependencies of the ROS packages.
+The `setup` task clones the repositories defined in `src/new_project.repos` and updates dependencies of the ROS packages.
 
-1. Run the `Tasks: Run Task` command in the
-   [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
-2. Select `setup` from the dropdown menu
+??? bug "Can't see the `setup` task"
 
-    ??? bug "Can't see the `setup` task"
+    If you can't see the `setup` task, run the `Developer: Reload Window` VS Code command.
+    This may occur when the workspace file is opened for the first time.
 
-        If you can't see the `setup` task, run the `Developer: Reload Window` command then rerun the `Tasks: Run Task` command.
-        This may occur when Sailbot Workspace is opened for the first time.
+## 8. Run the `Build All` task
+
+The `Build All` task builds all the ROS packages.
+
+## 9. Reload the VS Code terminals and window
+
+Delete all open terminals and run the `Developer: Reload Window` VS Code command
+to detect the files that were generated from building.
 
 ## Setup Sailbot Workspace in a GitHub Codespace
```

Comment:
Should we mark this section as optional?

Reply:
When you're building for the first time, the ROS and Python extensions won't detect your packages until you reload. Also you won't be able to run them in the terminals that were open before. Technically it's optional, but I thought it would be too in the weeds if I explained this and it's nice to have everything working.

Reply:
I mean setting up sailbot workspace in github codespaces

Reply:
Ah. Do you think I should collapse it just like I did with x11 forwarding?

Reply:
It doesn't have a number so it's not part of the setup instructions, just like troubleshooting isn't part of the workflow

Reply:
Either as a collapsible section or an optional step 10/section.

Reply:
I'm not sure if I want to number it, as it's not really part of the setup instructions, it is an alternate set up where you skip directly to step 6 after creating the codespace

Reply:
Made some changes, rerequesting a review